api/envoy/config/filter/network/rbac/v2/BUILD

```diff
@@ -5,5 +5,6 @@ licenses(["notice"])  # Apache 2
 api_proto_library_internal(
     name = "rbac",
     srcs = ["rbac.proto"],
+    require_py = False,
```

Comment:
FWIW, this is a pretty evil hack we added ages ago, I'd like to see it undone. Either we get rid of Python everywhere, or we support it.

Reply:
Can this be done separately? I can only take a certain amount of bazel hacking in a given week.

Reply:
Fixed